sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/ScenarioTests/DnsResolverTests.cs

```diff
@@ -0,0 +1,643 @@
+// ------------------------------------------------------------------------------------------------
+// <copyright file="DnsResolverScenarioTests.cs" company="Microsoft Corporation">
+//   Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------------------------
+
+namespace DnsResolver.Tests.ScenarioTests
+{
+    using FluentAssertions;
+    using Microsoft.Azure.Management.DnsResolver;
+    using Microsoft.Azure.Management.DnsResolver.Models;
+    using Microsoft.Azure.Management.Resources;
+    using Microsoft.Rest.Azure;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class DnsResolverScenarioTests : BaseScenarioTests
+    {
+        public DnsResolverScenarioTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void PutDnsResolver_ResolverNotExists_ExpectResolverCreated()
```

Comment:
Is checking cases such as limits breached (ex: # of resolver per subscription) in scope for E2E tests?

Especially given that customers will be aware that there is a limit.

Reply:
Resolve this one as we discussed.